src/Backoffice/Tasks/App/Controllers/CreateTaskController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Tasks\App\Request\UpsertTaskRequest;
+use Lightit\Backoffice\Tasks\App\Transformers\TaskTransformer;
+use Lightit\Backoffice\Tasks\Domain\Actions\StoreTaskAction;
+
+class CreateTaskController
+{
+    public function __invoke(UpsertTaskRequest $request, StoreTaskAction $storeTaskAction): JsonResponse
+    {
+        $task = $storeTaskAction->execute($request->toDto());
+
+      
+
+        return responder()
```

Comment:
Avoid leaving more than one space between lines:

```suggestion
        $task = $storeTaskAction->execute($request->toDto());

        return responder()
```